src/server/services/toeristische-verhuur.ts

```diff
@@ -87,8 +191,119 @@ export async function fetchToeristischeVerhuur(
   return apiSuccesResult(
     {
       registraties: registraties.content || [],
-      vergunningen: vergunningen.content || [],
+      vergunningen:
+        transformVergunningenToVerhuur(
+          vergunningen.content as VakantieverhuurVergunningen[]
+        ) ?? [],
+      daysLeft,
     },
     failedDependencies
   );
 }
+
+function createVergunningNotification(
+  item: VakantieverhuurVergunningen
+): MyNotification {
+  let title = 'Vakantieverhuur';
+  let description = 'Er is een update in uw vergunningsaanvraag.';
+  let datePublished = item.dateRequest;
+  let cta = 'Bekijk uw aanvraag';
+  let linkTo = item.link.to;
+
+  const monthsTillEnd = item.dateEnd ? monthsFromNow(item.dateEnd) : undefined;
+  const isActive = item.dateEnd ? new Date() < new Date(item.dateEnd) : false;
+  if (
+    item.caseType === 'B&B Vergunning' ||
+    item.caseType === 'Vakantieverhuur vergunningaanvraag'
+  ) {
+    switch (true) {
+      case item.status === 'Ontvangen' &&
+        isActualNotification(item.dateRequest, new Date()):
+        title = `Aanvraag vergunning gemeentelijke ${item.caseType}`;
+        description = `Wij hebben uw aanvraag voor een vergunning ${item.caseType} met gemeentelijk zaaknummer ${item.identifier} ontvangen`;
+        datePublished = item.dateRequest;
+        break;
+      case item.status === 'Afgehandeld' &&
+        isActualNotification(item.dateRequest, new Date()):
+        title = `Aanvraag vergunning gemeentelijke ${item.caseType}`;
+        description = `Wij hebben uw aanvraag voor een vergunning ${item.caseType} met gemeentelijk zaaknummer ${item.identifier} afgehandeld`;
+        datePublished = item.dateDecision || item.dateRequest;
+        break;
+      case isActive && monthsTillEnd && monthsTillEnd > -3:
```

Comment:
Magische nummers! `-3 en 3` kun je  die  in een constant  zetten met een expliciete naam `MONTHS_NEAR_END` oid?